public/app/plugins/panel/geomap/layers/data/utils.ts

```diff
@@ -0,0 +1,83 @@
+import { DataFrame, DataFrameView } from '@grafana/data';
+import { Point } from 'ol/geom';
+import { fromLonLat } from 'ol/proj';
+
+ /**
+   * Function that formats dataframe into a Point[]
+   */
+export function dataFrameToPoints(frame: DataFrame, config: any): Point[] {
```

Comment:
Should config have the GeomapPanelOptions type?

Reply:
Config is layer specific and can't have a more specific type. However, I've changed the input into the parts the function needs, so that the inputs can be more specific.